syntax = "proto3";

package suggestions;

service SuggestionsService {
    rpc GetSuggestions (SuggestionRequest) returns (SuggestionResponse);
}

message SuggestedBook {
    string bookId = 1;
    string title = 2;
    string author = 3;
}

message SuggestionRequest {
    string book_name = 1;
}

message SuggestionResponse {
    repeated SuggestedBook suggested_books = 1;
    bool approved = 2;
}